pkg/installer/installer.go

```diff
@@ -32,47 +31,7 @@ var (
 	uninstallerURL string
 )
 
-// runCommand executes a command and it's argurments in the CMD environment
-func runCommand(command string, arguments []string) string {
-	cmd := execCommand(command, arguments...)
-	var cmdOutput string
-	cmdReader, err := cmd.StdoutPipe()
-	if err != nil {
-		gorillalog.Warn("command:", command, arguments)
-		gorillalog.Warn("Error creating pipe to stdout", err)
-	}
-
-	var wg sync.WaitGroup
-	wg.Add(1)
-
-	scanner := bufio.NewScanner(cmdReader)
-	gorillalog.Debug("command:", command, arguments)
-	go func() {
-		gorillalog.Debug("Command Output:")
-		gorillalog.Debug("--------------------")
-		for scanner.Scan() {
-			gorillalog.Debug(scanner.Text())
-			cmdOutput = scanner.Text()
-		}
-		gorillalog.Debug("--------------------")
-		wg.Done()
-	}()
-
-	err = cmd.Start()
-	if err != nil {
-		gorillalog.Warn("command:", command, arguments)
-		gorillalog.Warn("Error running command:", err)
-	}
-
-	wg.Wait()
-	err = cmd.Wait()
-	if err != nil {
-		gorillalog.Warn("command:", command, arguments)
-		gorillalog.Warn("Command error:", err)
-	}
-
-	return cmdOutput
-}
+// cmd.RunCommand executes a command and it's argurments in the CMD environment
```

Comment:
This comment is orphaned and should either be moved or deleted.

Reply:
removed orphaned comment.